src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,107 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-family: $bodyFont, sans-serif;
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid $borderColor;
+  border-radius: 5px;
+
+  &__image {
+    margin: auto;
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-bottom: 40px;
+    background-size: 160px 134px;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text--big {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $textColorMain;
+  }
+
+  &__text--small {
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $textColorSecondary;
+  }
```

Comment:
Better to make a logic with modifications if you have one main class `&__text` and you want to change the font size (in your case)

But for now, you have different classes so better name it like elements

Reply:
Thanks for your feedback!